inabox/tests/setup_disperser_harness.go

```diff
@@ -101,7 +101,7 @@ type DisperserHarness struct {
 	// Controller components
 	// TODO: Refactor into a single struct for controller components
 	EncodingManager  *controller.EncodingManager
-	Dispatcher       *controller.Dispatcher
+	Dispatcher       *controller.Controller
```

Comment:
Consider also renaming this field. Not a requirement, depending on how much you want to expand scope

Reply:
renamed